### PR TITLE
feature/ADH-7539

### PR DESCRIPTION
**Pull request by mrhalless48GH**
https://tracker.yandex.ru/ADH-7539

## Why are the changes needed?

Close #xxx.

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: amoro-web/src/components/tables-sub-menu/TablesMenu.vue [hunk 1]
```diff
@@ -319,7 +321,14 @@ export default defineComponent({
...
           </div>
           <u-loading v-if="loading" />
-          <VirtualRecycleScroller :loading="loading" :items="databaseList" :active-item="database" :item-size="40" icon-name="database" @handle-click-table="handleClickDb" />
+          <VirtualRecycleScroller :loading="loading" :items="databaseList" :active-item="database" :item-size="40" @handle-click-table="handleClickDb">
+            <template #default="{ item }">
+              <svg-icon icon-class="database" class="table-icon g-mr-8" />
+              <p :title="item.label" class="name g-text-nowrap">
+                {{ item.label }}
+              </p>
+            </template>
```
Comment (asphix) [inline amoro-web/src/components/tables-sub-menu/TablesMenu.vue:330]: This looks like unnecessary code in parent components in the future. Perhaps it would be better to consider adding the database icon to tableTypeIconMap to get rid of the v-if-else and add it all to the default slot(s) (for the icon and content) so that you can override the content from the parent component for the whole component or just a specific part?
